client/controller/models/apps/apps.go

```diff
@@ -92,7 +92,7 @@ func Logs(c *client.Client, appID string, lines int) (string, error) {
 	u := fmt.Sprintf("/v2/apps/%s/logs", appID)
 
 	if lines > 0 {
-		u += "?log_lines=" + strconv.Itoa(lines)
+		u += "/?log_lines=" + strconv.Itoa(lines)
```

Comment:
yeah for some reason I had to add the slash in the client to get it to talk to the logger component. I'll look into seeing if I can fix this.